lib/src/store.rs

```diff
@@ -81,11 +81,24 @@ impl Store {
         write_locked_cache.insert(id.clone(), data.clone());
         Ok(data)
     }
-
     pub fn write_commit(self: &Arc<Self>, commit: backend::Commit) -> BackendResult<Commit> {
         assert!(!commit.parents.is_empty());
-        let commit_id = self.backend.write_commit(&commit)?;
-        let data = Arc::new(commit);
+        let mut mut_commit = commit;
+        let commit_id = loop {
+            // It's possible that another commit with the same commit id (but different
+            // change id) already exists. Fundge the timestamp until this is no
```

Comment:
Typo: Fundge -> Fudge (or "Slightly decrease")

Reply:
Thank you! I will un-fundge it.